homeassistant/components/lyric/__init__.py

```diff
@@ -76,6 +77,13 @@ async def async_update_data(force_refresh_token: bool = False) -> Lyric:
         try:
             async with asyncio.timeout(60):
                 await lyric.get_locations()
+            for location in lyric.locations:
+                for device in location.devices:
+                    async with asyncio.timeout(60):
+                        await lyric.get_thermostat_rooms(
+                            location.locationID,
+                            device.deviceID,
+                        )
```

Comment:
Instead of serializing the calls and timeouts, can we do one timeout and one gather of all the coroutines?